bootloader/src/pyi_utils.c

```diff
@@ -740,6 +740,26 @@ pyi_utils_set_environment(const ARCHIVE_STATUS *status)
     return rc;
 }
 
+/*
+ * If the program is actived by systemd socket, LISTEN_PID is set to parent
+ * pid of the bootloader and should be changed to child pid.
+ */
```

Comment:
Please put most parts of the commit message here, thus the reason for the code is easier to understand by just reading the code. Thanks.